setup.py

```diff
@@ -70,12 +70,12 @@ def get_include_path(library_name):
         return env_var_hint
 
     possible_directories = [
+        sysconfig.get_path('include')+f'/../{library_name}'
```

Comment:
I wonder if we should potentially find a more standard way for locating these, as this seems fragile. That might involve using CMake, although I'm not sure. But we can keep it if it is not likely to be fragile.

Reply:
I wonder what other projects do that use the `xsimd` and `eigen` conda packages.